src/js/inventory.js

```diff
@@ -4,6 +4,6 @@ export default (dependencies) => {
     setDependencies(dependencies);
 
     return import('./inventoryStyles').then(
-        () => import('@redhat-cloud-services/frontend-components-inventory')
+        () => import(/* webpackChunkName: "inventory-styles" */ '@redhat-cloud-services/frontend-components-inventory')
```

Comment:
I'd name this `inventory` instead of `inventory-styles`. Also might be resonable to name styles as well
```JS
return import(/* webpackChunkName: "inventory" */ './inventoryStyles').then(
  () => import(/* webpackChunkName: "inventory" */ '@redhat-cloud-services/frontend-components-inventory')
)
```